addons/udes_stock/data/locations.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
+<odoo>
+  <data>
+
+    <!-- Update "Warehouse/Stock" view location -->
+    <record id="stock.stock_location_stock" model="stock.location">
+      <field name="usage">view</field>
+    </record>
+
+
+    <!-- Construct "Warehouse/Input" view location -->
+    <record id="stock_location_input" model="stock.location">
+      <field name="name">Input</field>
+      <field name="location_id" ref="stock.warehouse0_view_location"/>
+      <field name="comment">Input zone</field>
```

Comment:
Nit: Can we uppercase `zone`? Also for output and received for `location_input_received`.

Thinks it looks nicer, i.e. like how we have `Goods Out Lane` on line 40.